drip/admin.py

```diff
@@ -55,26 +55,37 @@ def timeline(self, request, drip_id, into_past, into_future):
 
         return render(request, 'drip/timeline.html', locals())
 
+    def get_mime_html_from_alternatives(self, alternatives):
+        html = ''
+        mime = ''
+        for body, mime in alternatives:
+            if mime == 'text/html':
```

Comment:
Could we define 'text/hml' in a constant somewhere? You know, it could be hard if we should change it for some reason in the future.

Reply:
I only modularized the functions in order to decrease complexity. Didn't change anything. Maybe we can do another PR to improve constants?